src/stylesheets/layouts/_factors.scss

```diff
@@ -0,0 +1,145 @@
+.keyFactors {
+  @include section-setup( $color-mint , 4rem );
+  text-align: center;
+
+  &__wrapper {
+    position: relative;
+    @include flex;
+    width: 100%;
+    max-width: 77rem;
+    margin: 0;
+    padding: 2.5rem 1.5rem;
+
+    @media (min-width: 550px) {
+      margin: 2rem;
+      padding: 4rem 6.5rem;
+    }
+    @media (min-width: 800px) {
+      margin: 3rem;
+      padding: 5rem 8.5rem;
+    }
+  }
+
+  &__hdr {
+    padding-bottom: 1rem;
+  }
+
+  &__intro {
+    text-align: center;
+  }
+
+  &__list {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    flex-direction: column;
+
+    &__bodyimg {
+      width: 310px;
+      margin-left: auto;
+      margin-right: auto;
+      @media (min-width: 550px) {
+        width: 350px;
+        margin-top: 3.5rem;
+      }
+    }
+
+    &__hdr {
+      padding-bottom: 0.5rem;
+    }
+
+    &__content {}
+
+    &__item {
+      &--1 {
+        @include item;
+
+        &:before {
+          content: " ";
+          position: absolute;
+          display: block;
+          background-color: rgba(255, 255, 255, 0.63);
+          border-radius: 50px;
+          width: 60px;
+          height: 60px;
+          left: 44%;
+          top: -21%;
+        }
+
+        @media (min-width: 550px) {
+          max-width: 23rem;
+          padding: 1rem 1rem 1rem 2.5rem;
+          transform: translate(70%, 100%);
+          overflow-x: visible;
+
+          &:before {
+            content: " ";
```

Comment:
make these `content: "";` (nothing in between the quotes)

`content: " ";` will put a literal space into the element, like if it was a `<span> </span>` instead. This gets troublesome since that ` ` (space character) will affect the size of the psuedo element.

Reply:
resolved in pr#3